app/assets/javascripts/discourse/app/instance-initializers/codeblock-buttons.js

```diff
@@ -23,7 +23,12 @@ export default {
           return;
         }
 
-        const post = helper.getModel();
+        const post = helper.getModel?.();
```

Comment:
When can `helper` be missing `getModel` method?

Reply:
In the test for the copy codeblocks element. Do you have any idea why it would be missing that method?